sync/src/tests/chain.rs

```diff
@@ -159,7 +159,7 @@ fn propagate_hashes() {
 
 #[test]
 fn propagate_blocks() {
-	let mut net = TestNet::new(2);
+	let mut net = TestNet::new(20);
```

Comment:
Now the NewHashes message is sent out first, than NewBlock to some of the remaining peers. So this test requires more peers